src/arrayMethodSort.js

```diff
@@ -4,9 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultFunction) {
+    for (let n = 0; n < this.length; n++) {
+      for (let i = 0; i < this.length - 1 - n; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          [this[i], this[i + 1]] = [this[i + 1], this[i]];
+        }
+      }
+    }
+
+    return this;
   };
+
+  function defaultFunction(a, b) {
+    if (String(a) < String(b)) {
```

Comment:
On row 10 yout condition check if result of default compare function > 0, then perform some action. So check wich return -1 and 0 redundant, you can shorten them:
if (String(a) > String(b)) {
      return 1;
 }
 
 return 0